packages/main/src/CalendarHeader.js

```diff
@@ -53,6 +64,18 @@ const metadata = {
 		isMonthButtonHidden: {
 			type: Boolean,
 		},
+
+		isYearButtonHidden: {
+			type: Boolean,
+		},
+
+		/**
+		 * @type {boolean}
+		 * @private
+		 */
+		_isSecondaryCalendarType: {
+			type: Boolean,
```

Comment:
Delete this property from the metadata & use the created getter wherever needed

Reply:
Done